frontend/src/static/js/v_authorship.js

```diff
@@ -42,6 +42,7 @@ window.vAuthorship = {
       toReverseSortFiles: true,
       hasActiveFile: true,
       filterSearch: '*',
+      isBinaryFilesChecked: true,
```

Comment:
Can I check whats the general line number for a normal binary file (a picture)? If the line number if generally large then better set this to false.

Reply:
Not sure what you mean by line number? Currently I count their linecount as 0 and only show a single line of `Binary file not shown` no matter how large the binary file.

Reply:
I was wondering how the binary file will be sort according to LoC. So for your implementation they will all be placed at the bottom since all their line count is 0 in the json file?

Reply:
Yes sorting by line count will put them all at the bottom.